plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -52,7 +54,7 @@ describe('TaskTableUtil', function () {
       }
     });
 
-    this.sortFunction = TaskTableUtil.getSortFunction('name');
+    this.getComparator = TaskTableUtil.getSortFunction('name');
```

Comment:
I think we can still do this change in this PR. This helps readability of what is going on, but I do agree we should rename throughout the code base